### DISCUSSION TITLE
wallet: "No online backup file found" - what went wrong and what does it mean?

### DISCUSSION THREAD

**Discussion by ardabro** [Chia-Network/chia-blockchain#1753] [Q&A]
My farmer says it farms, but any operation on my wallet gives the following output:
```
(venv) #screen#>chia wallet get_address
No online backup file found, 
 Press S to skip restore from backup 
 Press F to use your own backup file:
```
Does it mean I don't have any address? So how it farms? Where is it going to put earned coins?
Last time due to some strange problems I removed `chia-blockchain` and `~/.chia` directories and reinstalled everything from scratch. I was surprised that despite my purge operation it found somewhere and reused my wallet keys. But apparently, something is wrong with this and it seems that chia operates without any addresses in my wallet.
How is it possible? How to solve it? I'm not sure if it is a bug but looks like a bug.

Output from farmer summary also gives a sinister message about wallet:
```
(venv) #screen#>cat summary.log 
Wed  7 Apr 23:35:06 CEST 2021
Exception from 'wallet' {'error': "'NoneType' object has no attribute 'tx_store'", 'success': False}
Farming status: Farming
```

**Comment by hoffmang9** [comment]
Just press S and the wallet will start syncing.

**Reply by almartin82** [reply]
Super helpful - can confirm that S will sync the wallet if two or more nodes are using the same keys.

**Comment by erikjber** [comment]
This happened to me. I pressed S and this was printed before the program closed:
```
Wallet height: 0
Sync status: Not synced
```
The same output is repeated whenever I re-run chia plotnft show.  

How long is it supposed to take? I've seen no change in 40 minutes.

**Reply by sfxworks** [reply]
Did you end up resolving this?

**Reply by erikjber** [reply]
I ended up deleting my installation including the database, reinstalling and recreating my wallet from my 24 words. 

Fortunately I was able to copy the database from another synced node so I did not need to do the full sync from scratch.  

Now it works.
